src/js/components/Select/SelectContainer.js

```diff
@@ -296,16 +326,8 @@ const SelectContainer = forwardRef(
           setActiveIndex(nextActiveIndex);
           setKeyboardNavigation(true);
         }
```

Comment:
Should we add a check here that if activeIndex === 0, the up arrow key will allow focus on the clear button?

Reply:
Done.

Reply:
Works nicely.